zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2550,6 +2588,42 @@ components:
           type: string
           description: The tenant id of the deployed form.
 
+    VariableRequest:
+      type: object
+      properties:
+        variables:
+          description: JSON object representing the variables to set in the element’s scope.
+          additionalProperties: true
+          type: object
+        local:
+          description: >
```

Comment:
```suggestion
          description: |
```
:x: I think we want to preserve the line-breaks in this comment. 

TIL [the difference between `|` and `>` for multi-line YAML](https://yaml-multiline.info/#block-scalars) 😄 


Here's what it looks like in a Swagger UI with `>` (and similar in the generated docs):

<img width="667" alt="image" src="https://github.com/user-attachments/assets/afe51aa5-24e1-41dd-ad48-dc450c3d9b3a">

And here's what it looks like with `|`:

<img width="668" alt="image" src="https://github.com/user-attachments/assets/5cd19b3d-37a3-4f16-9767-039814ba61fc">